src/build.py

```diff
@@ -218,10 +218,17 @@ def CopyLibraryToArchive(library, prefix=''):
   shutil.copy2(library, install_lib)
 
 
-def Tar(directory, print_content=False):
-  """Create a tar file from directory."""
+def Archive(directory, print_content=False):
+  """Create an archive file from directory."""
+  # Use the format "native" to the platform
   if not IsBuildbot():
     return
+  if IsWindows():
+    return Zip(directory, print_content)
+  return Tar(directory, print_content)
```

Comment:
Don't toy with me; I will add an unnecessary third argument to either `Zip` or `Tar` to avoid writing `return (Zip if IsWindows() else Tar)(directory, print_content)`